Game2048Base/src/main/java/models/grid2048/Grid2048.java

```diff
@@ -0,0 +1,117 @@
+package models.grid2048;
+
+import javalib.worldimages.*;
+import models.square.*;
+import utility.Utility;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.HashSet;
+
+public class Grid2048 {
+
+    public Square[][] grid;
+    static WorldImage IMAGE = new RectangleImage (
+            Square.SIDE_LENGTH * 4,
+            Square.SIDE_LENGTH * 4,
+            OutlineMode.OUTLINE,
+            Color.BLACK);
+
+    public Grid2048() {
+        this.grid = new Square[4][4];
+
+    }
+
+    public Grid2048(Square[][] grid) {
+        this.grid = grid;
+    }
+
+    public Grid2048 initializeStartingGrid() {
+        this.createEmptyTilesOnGrid();
+        this.addRandomTilesToGrid();
+
+        return this;
+    }
+
+    public void createEmptyTilesOnGrid() {
```

Comment:
I can't initialize a new starting grid by just calling the constructor because that would add two random tiles.
Do you suggest a better way to do this?